walkthroughs/eks/o11y-prometheus.md

```diff
@@ -57,5 +82,33 @@ Similarly, you can see all the scraped metrics (including application health met
 ## Cleanup
 
 ```
-helm delete appmesh-prometheus -n appmesh-system
+helm delete appmesh-prometheus --namespace appmesh-system
+```
+helm3 would support --purge by default. Do not follow the *Uninstalling the Chart* step in [appmesh-prometheus](https://github.com/aws/eks-charts/blob/master/stable/appmesh-prometheus/README.md)
```

Comment:
Is this because the instructions are incorrect in the appmesh-prometheus chart? If so lets fix them instead of calling them out as incorrect here.